### PR TITLE
Add more auto-generated files to .gitignore

### PR DESCRIPTION
**Pull request by glebm**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,10 @@
 htmldoc
 pkg
 webgen-tmp
+.bundle/
+CONTRIBUTERS
+VERSION
+kramdown.gemspec
+man/man1/kramdown.1
+node_modules/
+package-lock.json
```
Comment (gettalong) [inline .gitignore]: Lines 4, 9 and 10 have nothing to do with kramdown, please remove them. The other changes are fine.

Comment (glebm) [reply .gitignore]: Removed line 4.
Line 9 and 10 are for `npm install`:

https://github.com/gettalong/kramdown/blob/55d407290b62b1431e9657c9ee004b4fcc9f4bca/.travis.yml#L30

Comment (gettalong) [reply .gitignore]: That may be so but that line is from Travis and this is nothing that is generated by kramdown. As with .bundle, node_modules is something that the user decides, so it shouldn't be in the .gitignore file.

Comment (glebm) [reply .gitignore]: Done, though I disagree because this is the standard way of installing node dependencie.

**Comment by gettalong** [comment]
Merged

**Pull request closed without merging**